sample_configuration.properties

```
@@ -2,17 +2,17 @@ api.url={API_URL}
 database.url={DATABASE_URL}
 database.username={DATABASE_USERNAME}
 database.password={DATABASE_PASSWORD}
-institution.super_admin_user_one.email_address={INSTITUTION_SUPER_ADMIN_USER_ONE_EMAIL_ADDRESS}
-institution.super_admin_user_one.password={INSTITUTION_SUPER_ADMIN_USER_ONE_PASSWORD}
+afet_yonetim_sistemi_admin.email_address={AFET_YONETIM_SISTEMI_ADMIN_EMAIL_ADDRESS}
+afet_yonetim_sistemi_admin.password={AFET_YONETIM_SISTEMI_ADMIN_PASSWORD}
 institution_one.id={INSTITUTION_ONE_ID}
-institution_one.admin_user_one.email_address={INSTITUTION_ONE_ADMIN_USER_ONE_EMAIL_ADDRESS}
-institution_one.admin_user_one.password={INSTITUTION_ONE_ADMIN_USER_ONE_PASSWORD}
-institution_one.admin_user_two.email_address={INSTITUTION_ONE_ADMIN_USER_TWO_EMAIL_ADDRESS}
-institution_one.admin_user_two.password={INSTITUTION_ONE_ADMIN_USER_TWO_PASSWORD}
-institution.test_admin.email_address={INSTITUTION_TEST_ADMIN_EMAIL_ADDRESS}
-institution.test_admin.password={INSTITUTION_TEST_ADMIN_PASSWORD}
-landing.user_one.email_address={LANDING_USER_ONE_EMAIL_ADDRESS}
-landing.user_one.password={LANDING_USER_ONE_PASSWORD}
+volunteer_foundation_admin.email_address={VOLUNTEER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
+volunteer_foundation_admin.password={VOLUNTEER_FOUNDATION_ADMIN_PASSWORD}
+disaster_foundation_admin.email_address={DISASTER_FOUNDATION_ADMIN_EMAIL_ADDRESS}
+disaster_foundation_admin.password={DISASTER_FOUNDATION_ADMIN_PASSWORD}
+test_foundation_admin.email_address={TEST_FOUNDATION_ADMIN_EMAIL_ADDRESS}
+test_foundation_admin.password={TEST_FOUNDATION_ADMIN_PASSWORD}
+landing_page_user_martha.email_address={LANDING_PAGE_USER_MARTHA_EMAIL_ADDRESS}
+landing_page_user_martha.password={LANDING_PAGE_USER_MARTHA_PASSWORD}
```

@cagla88 Buraya kullanıcı ismi tanımlamak yerine yine kurum adıyla başlayarak `user` diyelim daha iyi olabilir. Örneğin `disaster_foundation.user.email_address` gibi tanımlayabiliriz.
```suggestion
landing_page_user_martha.email_address={LANDING_PAGE_USER_MARTHA_EMAIL_ADDRESS}
landing_page_user_martha.password={LANDING_PAGE_USER_MARTHA_PASSWORD}
```